notebooks/USE_embeddings.py

```diff
@@ -0,0 +1,220 @@
+# create USE embeddings for content from content store
```

Comment:
Some of this code has already been reviewed in this [PR](https://github.com/alphagov/govuk-content-similarity/pull/5)